### ISSUE TITLE
OutdatedDocumentation rejects aligned backtick-only parameter descriptions

### ISSUE THREAD

**Issue by TWiStErRob** [detekt/detekt#9527]
### Expected behavior

`OutdatedDocumentation` accepts aligned `@param` tags whose descriptions consist only of backticked text.

### Observed behavior

Only the first parameter is recognized and the remaining parameters are reported as undocumented:

```kotlin
/**
 * @param hue        `[0, 1]`
 * @param saturation `[0, 1]`
 * @param brightness `[0, 1]`
 * @param alpha      `[0, 1]`
 */
fun fromHsb(hue: Float, saturation: Float, brightness: Float, alpha: Float) = Unit
```

The finding is:

```text
Documentation of fromHsb is outdated: parameters 'saturation', 'brightness', 'alpha' are not documented
```

Removing the alignment spaces does not help. Replacing the backtick-only descriptions with prose such as `the saturation in the [0, 1] range` makes the finding disappear.

### Context

- Detekt: `2.0.0-alpha.5`
- `allRules = true`
- Type-resolved Android/Kotlin analysis

The aligned form was accepted by Detekt 1.23.8.
